index.html

```diff
@@ -0,0 +1,314 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <title>Web Dev Course (U.O.M)</title>
+    <body>
+
+        <header class="mobile_nav">
+          <input type="checkbox" class="nav-checkbox" id="nav-switch" />
+          <label for="nav-switch" class="nav_btn">
+            <span class="nav_icon">&nbsp;</span>
+          </label>
+          <div class="nav_block">&nbsp;</div>
+          <nav class="nav_content">
+            <ul class="mobile_content_links">
+              <li class="mobile_link">
+                <a href="./about.html" class="mobile_link_items">About</a>
+              </li>
+              <li class="mobile_link">
+                <a href="#main" class="mobile_link_items">Program</a>
+              </li>
+              <li class="mobile_link">
+                <a href="#join" class="mobile_link_items">Join</a>
+              </li>
+              <li class="mobile_link">
+                <a href="#sponsor" class="mobile_link_items">Sponsor</a>
+              </li>
+              <li class="mobile_link"><a href="#" class="mobile_link_items">News</a></li>
+              <li class="mobile_link">
+                <a href="#" class="mobile_link_items">CC Campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </header>
+
+        <header class="header">
+          <nav class="authors_social_link">
+            <ul class="authors_profile_container">
+              <li class="authors_profile">
+                <a href="#" class="authors_profile_link"><img src="./images/facebook-square-brands.jpg" class="social_mid" alt="facebook_profile"></a>
+              </li>
+              <li class="authors_profile">
+                <a href="#" class="authors_profile_link"><img src="./images/twitter-square-brands.jpg" class="social_mid" alt="twitter_profile"></a>
+              </li>
```

Comment:
Let's fix the social media logo as requested by the previous code reviewer ✊
![Screenshot (312)](https://user-images.githubusercontent.com/66112760/142586421-a8b4ae1c-68f9-492f-aab6-7786f8e3e27f.png)